src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -373,23 +374,23 @@ const EditHomepage = ({ match }) => {
             textCardItemErrors.push(
               _.map(cards, () => getErrorValues(TEXTCARDS_ITEM_SECTION))
             )
-            if (textCardItemErrors.length === 0) {
-              textCardItemErrors.push([])
-            }
```

Comment:
what happens now when there are no items?

Reply:
Replied below - this logic is to basically maintain sectionIdx - errors mapping